packages/core/src/components/portal/portal.tsx

```diff
@@ -75,84 +74,82 @@ const PORTAL_LEGACY_CONTEXT_TYPES: ValidationMap<PortalLegacyContext> = {
  *
  * @see https://blueprintjs.com/docs/#core/components/portal
  */
-export function Portal(props: PortalProps, legacyContext: PortalLegacyContext = {}) {
+export function Portal(
+    { className, stopPropagationEvents, container, onChildrenMount, children }: PortalProps,
+    legacyContext: PortalLegacyContext = {},
+) {
     const context = React.useContext(PortalContext);
 
-    const [hasMounted, setHasMounted] = React.useState(false);
+    const portalContainer = container ?? context.portalContainer ?? document?.body;
```

Comment:
can you please update the Portal docs to reflect this behavior? Specifically, these lines should change: https://github.com/palantir/blueprint/blob/8623da5404bc1be186f696aca0a92a41bd1db5d9/packages/core/src/components/portal/portal.md#L10-L11

- Let's remove outdated references to JQuery
- Let's mention that the new portal element is added as a child of either (in order of preference): a `container` element specified as a `<Portal>` prop, a `portalContainer` element specified via `<PortalProvider>` context, or `document.body`

Also, I noticed a broken link on the Portal docs page. This line should be updated: https://github.com/palantir/blueprint/blob/8623da5404bc1be186f696aca0a92a41bd1db5d9/packages/core/src/components/portal/portal.md#L34

The correct link is `#core/context/portal-provider`